### ISSUE TITLE
List of future cookbooks

### ISSUE THREAD

**Issue by jdannberg** [geodynamics/aspect#2110]
It would be good to have a list of things that need more documentation, so that people who have experience with that part of ASPECT could write a cookbook when they have the time (it could be similar to the starter projects). So let's compile a list of cookbooks we want to do in the future. Please add to this list whatever you think we need! I think @gassmoeller had a number of things in mind that need more documentation...

- [ ] cookbook using thermodynamically consistent data (#2063) 
- [ ] subduction cookbook (#1828)
- [ ] Stabilization (#1039)
- [ ] Particles (#942)
- [ ] dG method (#725)
- [ ] mid-ocean ridge cookbook

**Comment by ljhwang** [comment]
I recall starting a  “network” diagram at last year’s hack that I think was suppose to aid in this issue.

Best,
-Lorraine

*****************************
Lorraine Hwang, Ph.D.
Associate Director, CIG
530.752.3656

**Comment by jdannberg** [comment]
Yes, that would definitely be helpful for this, thanks for reminding us, @ljhwang!
I recall that @bangerth should have all the code/data/output of what you did. @bangerth, can we somehow get that into the manual?

**Comment by arijitlaik** [comment]
A simple Accretionary wedge cookbook would definitely help me out, with boundary conditions like this(in 2d):
![ruh2013](https://user-images.githubusercontent.com/10157121/36627112-f08c7bbe-1963-11e8-9deb-79889873d304.png)

**Comment by naliboff** [comment]
@arijitlaik - That is from the Ruh et al. 2013 G3 paper, yes? We can certainly try setting up this problem in 2D and 3D.  Would you like to discuss this in a separate "issue" or offline initially? It may be easier to use a free surface verse sticky air, but other than that I don't see any reason this can't be done. Not having a moving wall makes the problem much easier.

**Comment by naliboff** [comment]
@jdannberg - Thanks for starting this list. Another cookbook that would a number of people have requested is a post-glacial rebound type problem. This should be straightforward after viscoelasticity is finished.

**Comment by arijitlaik** [comment]
@naliboff its from the Ruh et al. 2013 G3, i should have mentoned it there, i have been trying to model these types of geometries and have been fairly successful in implementing them in Underworld(sticky air, without surface diffusion) and in SiStER(sticky air with linear surface difusion), I would definitely like to be in loop with constructing these types of models and share my already constructed models, in an issue or via email laikarijit_at_gmail_dot_com.

**Comment by naliboff** [comment]
@arijitlaik - great, let's take the discussion off github until we have a few concrete results to report back with. There may be a few rounds of trying various things to get reasonable convergence behavior. If others would like the full discussion to take place via a new issue, happy to do so and please let us know.

**Comment by bobmyhill** [comment]
This issue now has a new project page: https://github.com/orgs/geodynamics/projects/15
